### ISSUE TITLE
UPDATION IN README.md FILE REQUIRED 

### ISSUE THREAD

**Issue by soham0028** [Vanshikapandey30/Hacktoberfest2024#179]
Hello @Vanshikapandey30  
The current readme file does not contain the following :
1. table of contents for easy access 
2. back to top button for easy navigation
3. contributors section
4. forkers and stargazers section 
5. featured in section 

Kindly assign me the issue with hacktober fest labels so that my work is counted for the event

**Comment by Sar-taj107** [comment]
Yes, @soham0028 correctly pointed out that the current README.md file in the repository lacks proper structure. Please assign this issue to @Vanshikapandey30 under the Hacktoberfest tag. We are excited to collaborate on this and  look forward to improving it together! to make it more easy to accessible for all.
Thanks!
